src/distribution.cpp

```diff
@@ -381,14 +362,16 @@ UPtrDist distribution_from_xml(pugi::xml_node node)
     dist = UPtrDist {new Watt(node)};
   } else if (type == "normal") {
     dist = UPtrDist {new Normal(node)};
-  } else if (type == "muir") {
-    dist = UPtrDist {new Muir(node)};
   } else if (type == "discrete") {
     dist = UPtrDist {new Discrete(node)};
   } else if (type == "tabular") {
     dist = UPtrDist {new Tabular(node)};
   } else if (type == "mixture") {
     dist = UPtrDist {new Mixture(node)};
+  } else if (type == "muir") {
+    openmc::fatal_error(
+      "'muir' distribution type is no longer supported. Please regenerate your "
+      "XML files using the Python API.");
```

Comment:
Perhaps different wording but what do you think about adding a version number

```suggestion
    openmc::fatal_error(
      "'muir' distribution type is no longer supported (v0.14.0 and above). "
      "Please regenerate your XML files using the Python API.");
```

Reply:
I'll make a suggestion to the suggestion 😁 

```suggestion
    openmc::fatal_error(
      "'muir' distributions are now specified using normal distributions (v0.14.0 and above)."
      "Please regenerate your XML files using the Python API.");
```

While we understand that the Muir distribution is a normal distribution with parameters defined in certain manner, this might not be obvious to others

Reply:
That looks perfect to me, thanks @pshriwise

Reply:
A user won't have to manually specify a normal distribution, so that message seems misleading to me (they should be using the `muir` function). Also, adding the version doesn't really add any useful information -- if the user gets this message, they _are_ using version v0.14.0 or above.

Reply:
Fair point about the version number. Regarding the wording around the distribution, the main thing I was trying to avoid is an interpretation that Muir distributions aren't supported at all anymore.

Reply:
haha yes ok fair enough, I was perhaps planning for users with time machines :rofl: